jwst/pipeline/calwebb_spec3.py

```diff
@@ -20,6 +20,7 @@
 # Group exposure types
 MULTISOURCE_MODELS = ['MultiSlitModel']
 IFU_EXPTYPES = ['MIR_MRS', 'NRS_IFU']
+SLITLESS_TYPES = ['NIS_WFSS', 'NRC_WFSS'] #, 'NRC_TSGRISM']

```

Comment:
NRC_TSGRISM is already comment out so it has no effect on anything, but we really could just completely remove it, because time-series grism data should never go through this pipeline anyway. They go through ``calwebb_tso3``.